### ISSUE TITLE
cannot produce the results in table 4

### ISSUE THREAD

**Issue by seyeeet** [baidut/PaQ-2-PiQ#3]
Hello
Thank you for the great work.
I am trying to re-produce the results in table 4 of the paper however I cannot. 
![image](https://user-images.githubusercontent.com/62394093/105606923-f3e8c680-5d69-11eb-9b46-1b320beabd7b.png)

- should we test the model only on the validation set of CLIVE and KonIQ or on the whole dataset?

- can you share the trained models,
P2P-BM (Sec. 4.1) , P2P-RM (Sec. 4.2), and P2P-FM (Sec. 4.4), that can produce those results?

**Comment by baidut** [comment]
I found the original code to reproduce the results, will provide a google colab notebook and get back to you in 1 or 2 days.

**Comment by baidut** [comment]
Please follow [this google colab](https://colab.research.google.com/drive/1M2sW86mCBMRkYFa80tI4M85cP8Nrw_kI?usp=sharing) to reproduce the results. Thanks!

**Comment by Jian-Yin-Shine** [comment]
> Please follow [this google colab](https://colab.research.google.com/drive/1M2sW86mCBMRkYFa80tI4M85cP8Nrw_kI?usp=sharing) to reproduce the results. Thanks!

Hello, which data set is this model parameter trained from? Is it a larger controversial data set than that in the paper?
